File: src/main/kotlin/dev/jefy/connectpro/messaging/domain/repositoty/ConversationRepository.java
package dev.jefy.connectpro.messaging.domain.repositoty;


import org.springframework.stereotype.Repository;

import dev.jefy.connectpro.messaging.domain.Conversation;
import dev.jefy.connectpro.messaging.domain.vo.ConversationId;
import dev.jefy.connectpro.shared.infrastructure.ddd.AggregateRepository;

/**
 * @author Jôph Yamba
 */
@Repository
public interface ConversationRepository extends AggregateRepository<Conversation, ConversationId> {
}

File: src/main/kotlin/dev/jefy/connectpro/messaging/domain/repositoty/ConversationRepository.kt
package dev.jefy.connectpro.messaging.domain.repositoty


import dev.jefy.connectpro.messaging.domain.Conversation
import dev.jefy.connectpro.messaging.domain.vo.ConversationId
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * @author Jôph Yamba
 */
@Repository
interface ConversationRepository : JpaRepository<Conversation, ConversationId>

